src/main/kotlin/no/nav/familie/ef/sak/api/gui/VurderingController.kt

```diff
@@ -14,21 +14,20 @@ import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import java.util.*
 
-
 @RestController
 @RequestMapping(path = ["/api/vurdering"], produces = [MediaType.APPLICATION_JSON_VALUE])
 @ProtectedWithClaims(issuer = "azuread")
 @Validated
 class VurderingController(val vurderingService: VurderingService) {
 
-    @GetMapping("{sakId}/medlemskap")
-    fun getMedlemskapVurdering(@SakstilgangConstraint @PathVariable sakId: UUID): Ressurs<MedlemskapDto> {
-        return Ressurs.success(vurderingService.vurderMedlemskap (sakId))
+    @GetMapping("{sakId}/inngangsvilkaar")
```

Comment:
Jeg har lagt opp ordbøkene for team familie med substitusjonene å -> a, ø -> o og æ -> e. Hvis vi holder oss til det, så slipper vi å legge til så mange ord manuelt.

Reply:
ff584e0